### ISSUE TITLE
Problem: if hax has crashed, neither 'hctl shutdown' nor 'hctl bootstrap' will work

### ISSUE THREAD

**Issue by knekrasov** [Seagate/cortx-hare#1498]
JIRA: https://jts.seagate.com/browse/EOS-17407

The problem was observed during Mini Provisioner integration session.

When hax crashed (by whatever reason), neither of hctl scripts could help to get the Hare cluster to some meaningful state. We could neither shut it down ("Cluster is not running!") nor bootstrap again ("hare-consul-agent service is found => cluster is running"). 

The reason is that two different scripts (`hare-bootstrap` and `hare-shutdown`) use different criteria of "Hare cluster is running". 
https://github.com/Seagate/cortx-hare/blob/3239e6c418f3635e75c3e0c4d04b017f7b5aacd1/utils/hare-bootstrap#L212-L214

https://github.com/Seagate/cortx-hare/blob/3239e6c418f3635e75c3e0c4d04b017f7b5aacd1/utils/hare-shutdown#L207-L211

## Solution
* Use the same criteria in both scripts - no matter which one.

**Comment by mssawant** [comment]
We need to handle this asap, I just hit an issue where I triggered `hctl shutdown`, cancelled it in between  so motr processes did not stop. I tried `hctl shutdown` again and it says `cluster is not running` while m0d process was running.
```
[root@ssc-vm-1623:cortx-hare-1] hctl shutdown
Cluster is not running
[root@ssc-vm-1623:cortx-hare-1] ps -aux | grep m0d
root      29433  0.0  0.0 112808   968 pts/0    S+   09:58   0:00 grep --color=auto m0d
root      84865  0.0  2.0 13977424 326688 ?     SLsl Feb07   0:39 /usr/bin/m0d -e lnet:10.230.250.51@tcp:12345:2:1 -A linuxstob:addb-stobs -f <0x7200000000000001:0x9> -T linux -S stobs -D db -m 65536 -q 16 -c /etc/motr/confd.xc -H 10.230.250.51@tcp:12345:1:1 -U -r 1073741824
```

**Comment by knekrasov** [comment]
I re-read my own description and it feels like I mislead myself. Title of this issue is incorrect.

The predicates are effectively the same, so the problem is different.

1. Both bootstrap and shutdown look at Consul to understand whether the cluster is running. Only `hctl status` is different: https://github.com/Seagate/cortx-hare/blob/3239e6c418f3635e75c3e0c4d04b017f7b5aacd1/utils/hare-status#L202-L207
But it doesn't look like an issue.
2. `hctl shutdown` depends heavily on the existence of Consul. Without Consul KV it will  not know which processes it must stop. So it is critical that Consul must be stopped after all other processes, and according to the sources, it acts as required 🤔 

@mssawant Can it happen that consul agent was stopped before m0d processes at your machine? If you're able to reproduce the issue, can you attach the output of `hctl shutdown` please?

**Comment by knekrasov** [comment]
Closed in scope of #1503
